CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## vNEXT
+- Update requester/voucher debits to refund on claims amounts divisible by volume. (#28)
```

Comment:
```suggestion
- Make sponsored & non-sponsored amounts always divisible by deal volume to refund tasks fairly. (#28)
```

Reply:
I like this version. We can also do:
```suggestion
- Make sponsored & non-sponsored amounts always divisible by deal volume to avoid slippage in later refunds. (#28)
```

Reply:
Ok, are you sure we want to use the word _slippage_, which has a dedicated (and other) meaning in token and finance :thinking:

Reply:
We can skip using it here if it's confusing.
But I find it suitable for this case because the refunded amount slips away from the original paid amount (for the voucher or the requester.

![image](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/assets/26070035/497a9843-0ace-411a-b14b-b00543725a8e)